Office 365/o365/ingest/parser.yml

```diff
@@ -136,6 +302,60 @@ stages:
         mapping:
           event.action: event.type
         fallback: '["info"]'
+      - set:
+          office365.exchange.item.parent_folder: "{{json_event.message.Item.ParentFolder.Path}}"
+        filter: '{{json_event.message.get("Item") != None and json_event.message.Item.get("ParentFolder") != None}}'
+      - set:
+          email.message_id: "{{json_event.message.Item.InternetMessageId[1:-1]}}"
+          email.subject: "{{json_event.message.Item.Subject}}"
+        filter: '{{json_event.message.get("Item") != None and json_event.message.Item.get("InternetMessageId") != None}}'
+
+  parse_exchange_item_group:
+    actions:
+      - set:
+          email.message_id: >
+            [
+              {% for item in json_event.message.AffectedItems %}
+                {% if item.get("InternetMessageId") != None %}
+                  "{{item.InternetMessageId[1:-1]}}",
+                {% endif %}
+              {% endfor %}
+            ]
+          email.subject: >
```

Comment:
[`email.subject`](https://www.elastic.co/guide/en/ecs/8.8/ecs-email.html#field-email-subject) doesn't expect a list as value.
Maybe a custom field is more suitable